### PR TITLE
User info

### PR DESCRIPTION
**Pull request by paultgibbons**
@joshuacamp I was having issues with finding the redactors remove all methods, so I commented them out in the first commit. Do you know whats going on

Other than that, just addresses the issues that are assigned to me

### CODE REVIEW

**Comment by joshuacamp** [comment]
@paultgibbons Can you give me a more detailed explanation of what errors you're getting when calling the `removeAllRelations()` method?

File: core/app/models/BaseViewModel.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package models;
+
+public class BaseViewModel {
+
+    public String user;
```
Comment (joshuacamp) [inline core/app/models/BaseViewModel.java]: Do you think we should have constructors for the `ViewModel` classes? I feel like not having them could cause some minor problems for people working on the project in the future. But again, I'm not sure about the conventions of the view model/ scala templating stuff so it's fine if this is how it's typically done.

**Comment by paultgibbons** [comment]
@joshuacamp I cant reproduce it, pushed a commit to undo it.
Also made Base View Model abstract and with its own constructor. I think that should be fine for now

**Comment by joshuacamp** [comment]
@paultgibbons Nice. The `removeAllRelations` method was added in the last update to the cogcomp libraries so that might've been it. I'll merge when travis passes.

**Pull request merged**